source/Engine.h

```diff
@@ -135,7 +135,7 @@ class Engine {
 		Point center;
 		Angle angle;
 		double radius;
-		int type;
+		const Color &color;
```

Comment:
Not sure why was this changed? I dont see this new variable being used but mb I missed it

Reply:
I changed this so each set of target pointers (this class, Engine::Target holds the parameters for the target pointers) can have it's own colour instead of just refering to one of the Radar types.
It is used in Engine::Draw() when drawing the target pointers where the shadar is passed `target.color` instead of `Radar::GetColor(target.type)`.

Reply:
oh right engine holds the target pointer, thx MZ
alr seems good